targets/targets.json

```diff
@@ -2364,7 +2365,8 @@
             "MPU"
         ],
         "release_versions": ["2", "5"],
-        "device_name": "STM32F401RE"
+        "device_name": "STM32F401RE",
+        "components_add": ["ESP8266"]
```

Comment:
Can't be added in the default targets json configuration!

Reply:
i've added the component in targets.json only because those platforms were configured specially in the mbed_lib.json